rpmlb/downloader/base_rpkg.py

```diff
@@ -22,8 +22,9 @@ def download(self, package_dict, **kwargs):
 
         package = package_dict['name']
         cmd = r'''
-{} co {} && \
-cd {} && \
-git checkout {}
-        '''.strip().format(self.command, package, package, branch)
+{command} co {package} && \
+cd {package} && \
+git checkout {branch} && \
+{command} sources
+        '''.strip().format(command=self.command, package=package, branch=branch)
```

Comment:
> E501 line too long (80 > 79 characters)

Reply:
OK, I will wrap that line.

That reminds me that we should probably add a project flake8 configuration, since mine ignore line length warnings (compatibility with [black](https://github.com/ambv/black)) – so this test passes on my machine :)